frontend/src/hooks/useChangeRequests.hooks.js

```diff
@@ -0,0 +1,59 @@
+import { useGetAgreementByIdQuery, useGetCansQuery } from "../api/opsAPI";
```

Comment:
this is the crux of the PR. This hook sends off an array of properly formatted changeRequests to the AgreementChangeRequestAlert